clang/utils/analyzer/exploded-graph-rewriter.py

```diff
@@ -1186,6 +1192,14 @@ def main():
         "displaying it, dump the rewritten dot file "
         "to stdout",
     )
+    dump_conflict.add_argument(
+        "--dump-html-only",
+        action="store_const",
+        dest="dump_html_only",
+        const=True,
+        default=False,
+        help="do not open the generated HTML immediately",
```

Comment:
I was expecting this comment to be fairly similar to the one provided for `--dump-dot-only`.
Why is it different?

Reply:
As it is presented directly after `--dump-dot-only`, I used to think the help message is enough to understand the option after reading the `--dump-dot-only`'s. Updated as suggested.